src/test/java/catan/controllers/testcases/game/ListGameTest.java

```diff
@@ -234,6 +234,6 @@ public void should_successfully_get_joined_games_list_even_if_user_is_not_creato
                 .post(URL_CURRENT_GAMES_LIST)
                 .then()
                 .statusCode(200)
-                .body("findall.size()", is(2));
+                .body("findAll.size()", is(2));
```

Comment:
why do I need to add incorrect word to library if it's better to fix the typo?
Here is Rest-Assured Wiki, you can find how is correct by Ctrl+F searching:
https://github.com/jayway/rest-assured/wiki/Usage#xml-example